lib/thumbnail-sizes.php

```diff
@@ -25,6 +25,8 @@
   add_image_size( 'l-8-square', 866, 866, true );
   add_image_size( 'l-8-full', 866, 9999, false );
 
+  add_image_size( 'lightbox', 1030, 644, false );
```

Comment:
I don't remember cuz it didn't fit the screen vertically, neither on my monitor nor in the macbook screen. So the image would almost always end up scaling down anyway. 

I used 10 col width.